minetest/minetest
# Add an option to disable unittest build, & disable them on Docker build

All is in the title.
It's not always useful to build unittests, especially on the docker build and on release builds (maybe).
Add a switch for that

## To do

This PR is Ready for Review.

## How to test
build the project with -DBUILD_UNITTESTS=0 and see that the unittest flag disappear from our command line
